src/controllers/metadata.controller.ts

```diff
@@ -68,9 +68,8 @@ export class MetadataController extends Controller {
                 }
             });
 
-            let mapped = <any>resp;
-            mapped.hits.hits = resp.hits.hits.map(h => h._source);
-            return mapped.hits;
+            const results = resp.body.hits.hits.map((hit: any) => hit._source)
+            return results.hits;
```

Comment:
In this file there is more work to be done.

Fixes with index name like here: https://github.com/ServerCentral/elastalert-server/commit/63bcc53b1ca1fea3024d6b6fa01943f6a6d3f728#diff-32d34ef54e6d3c4ee1cc0c37f9a0aa2e28c6b75c2c0a74ba46337e93df45da2a
I will fix it in separate commit.